File: challenges/6-converter-segundos/python/scush989898/desafio-6.py
seconds = int(input())


def pad_start(n: int) -> str:
    if n >= 10:
        return str(n)
    return "0" + str(n)


hours = seconds // 3600
seconds -= hours * 3600

minutes = seconds // 60
seconds -= minutes * 60

formated = f"{pad_start(hours)}:{pad_start(minutes)}:{pad_start(seconds)}"

print(formated)

File: challenges/6-converter-segundos/javascript/codinghobbit/desafio6.js
const readline = require("readline").createInterface({
  input: process.stdin,
  output: process.stdout,
});

readline.question("Insira os segundos: ", (sec) => {
  const hours = parseInt(sec / 3600);
  const minutes = parseInt((sec % 3600) / 60);
  const seconds = parseInt(sec % 60);
  console.log(
    `${hours < 10 ? "0" + hours : hours}:${
      minutes < 10 ? "0" + minutes : minutes
    }:${seconds < 10 ? "0" + seconds : seconds}`
  );
  readline.close();
});
